inference-engine/src/inference_engine/src/cpp_interfaces/interface/ie_iplugin_internal.cpp

```diff
@@ -219,6 +225,85 @@ void IInferencePlugin::SetExeNetworkInfo(const std::shared_ptr<IExecutableNetwor
     // Set inputs/outputs and pointer to plugin manually here
     exeNetwork->setNetworkInputs(copyInfo(constMapCast(inputs)));
     exeNetwork->setNetworkOutputs(copyInfo(constMapCast(outputs)));
+
+    ngraph::ParameterVector parameters;
+    ngraph::ResultVector results;
+    std::vector<ngraph::Output<ngraph::Node>> node_outputs;
+
+    for (auto&& input : inputs) {
+        auto tensor_desc = input.second->getTensorDesc();
+        auto dims = tensor_desc.getDims();
+        parameters.push_back(std::make_shared<ngraph::op::v0::Parameter>(
+            details::convertPrecision(tensor_desc.getPrecision()),
+            ngraph::PartialShape{std::vector<ngraph::Dimension>{dims.begin(), dims.end()}}));
+        parameters.back()->set_friendly_name(input.first);
+        node_outputs.push_back(parameters.back()->output(0));
+    }
+
+    auto node = std::make_shared<ExecGraphInfoSerialization::ExecutionNode>(node_outputs, outputs.size());
+
+    int i = 0;
+    for (auto&& output : outputs) {
```

Comment:
Maybe can we  use the loop from 0 to outputs.size()?

Reply:
`outputs` is a `std::map` here